Blaise-Shyaka/Politico
# [#171591706] Create user sign up page

# What does this PR do?

It creates a page allowing a user to create a new account

# Description of tasks to be completed

- Create a form where a user can fill in personal details
- Make the page attractive, simple and responsive

# How should this be manually tested?

- Clone this repository and open `/templates/signup.html`
- Check for the presence of a form prompting a user to fill in personal details
- Resize the browser to verify its responsiveness

# What are the relevant pivotal tracker stories?

#171591706

# Screenshots

![Screenshot (16)](https://user-images.githubusercontent.com/52489421/75836116-80dd0700-5dc9-11ea-978c-f42ace69f6be.png)
![Screenshot (14)](https://user-images.githubusercontent.com/52489421/75836118-84708e00-5dc9-11ea-941c-aaacdd8952e7.png)
![Screenshot (15)](https://user-images.githubusercontent.com/52489421/75836120-85092480-5dc9-11ea-922b-6d076037d100.png)
